app/assets/stylesheets/shared/header.scss

```diff
@@ -120,11 +127,12 @@ header.navigation {
     a {
       color: $navigation-color;
       display: inline-block;
-      font-weight: 400;
+      font-weight: bold;
```

Comment:
Properties should be ordered color, display, font-size, font-weight, text-decoration